### PR TITLE
A bunch of minor markups

### PR DESCRIPTION
**Pull request by robbrockbank**

### CODE REVIEW

File: master/reference/advanced/overlap-ips.md [hunk 1]
```diff
@@ -66,6 +66,7 @@ Routing namespaces may be needed, as the compute host may have VMs using the sam
 To route translated packets across the network between compute hosts, BGP must distribute IPv6-translated addresses for instances, instead of the original IPv4 addresses.
 
 So the picture for processing a packet from a VM looks like this:
+
```
Comment (caseydavenport) [inline master/reference/advanced/overlap-ips.md]: I've already got a PR open for this one.

File: master/reference/calicoctl/resources/policy.md [hunk 2]
```diff
@@ -48,7 +48,7 @@ spec:
 
 | Field    | Description                 | Accepted Values   | Schema | Default    |
 |----------|-----------------------------|-------------------|--------|------------|
-| order    | Indicates priority of this policy, with lower order taking precedence. | | integer | 100 |
+| order    | Indicates priority of this policy, with lower order taking precedence.  No value indicates highest order (lowest precedence) | | float | unassigned (highest order) |
```
Comment (gunjan5) [inline master/reference/calicoctl/resources/policy.md]: Not sure if I understand the Default value field `unassigned (highest order)` does it mean it's unassigned by default? or `unassigned highest order`

File: master/reference/calicoctl/resources/profile.md [hunk 3]
```diff
@@ -15,8 +15,9 @@ aliases are supported (all case insensitive): `profile`, `profiles`, `pro`, `pro
 
 ### Sample YAML
 
-The following sample profile allows all traffic from 10.0.0.0/16, except if the source 
-is within 10.244.0.0/24, or if the source is within the same profile.
+The following sample profile allows all traffic from endpoints that
+have the profile label set to profile1 (i.e. endpoints that reference this profile),
```
Comment (gunjan5) [inline master/reference/calicoctl/resources/profile.md]: can you add a `'` around `profile1` in indicate it's a name or make it a code by adding ` around it

File: master/reference/calicoctl/resources/policy.md (hunk 2 above)
Comment (robbrockbank) [inline master/reference/calicoctl/resources/policy.md]: by default it's unassigned, which is treated as "highest order"

**Pull request merged**